modules/network-observability-ebpf-rule-flow-filter.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="network-observability-ebpf-flow-rule-filter_{context}"]
+= eBPF flow rule filter
```

Comment:
and should this be `ebpf flow filter rule`? It it a typo or thats what was suggested?

Reply:
I got this from the upstream doc: https://github.com/netobserv/netobserv-ebpf-agent/blob/f2a65b3639d01c06092555312c2b85b43881aa5d/docs/flow_filtering.md#ebpf-flow-rule-based-filtering

The title here is `eBPF Flow Rule Based Filtering`

Reply:
Should it be different 🤔

Reply:
oh then okay